### PR TITLE
fix: Update Auth to import JS using named export 

### PR DESCRIPTION
**Pull request by jamesaucode**
#### Description of changes
Using named export instead of the default export from JS to reduce bundle size

#### Issue #, if available
N/A

#### Description of how you validated changes
`yarn build && yarn test`

#### Checklist
- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ashika01 — APPROVED** [review]
LGTM. Was there a reason around why we left this around? @elorzafe

**Review by reesscot — APPROVED** [review]
LGTM!

**Comment by jamesaucode** [comment]
> @jamesaucode tests are failing

Yea, the hostedUI test is mocking the JS module, I will fix it in the next commit.

File: packages/auth/__tests__/hosted-ui.test.ts [hunk 1]
```diff
@@ -157,7 +157,8 @@ jest.mock('amazon-cognito-identity-js/lib/CognitoUser', () => {
 	return CognitoUser;
 });
 
-import { Hub, Credentials, StorageHelper, JS } from '@aws-amplify/core';
+import * as AmplifyCore from '@aws-amplify/core';
+const { Hub, Credentials, StorageHelper } = AmplifyCore;
```
Comment (jamesaucode) [inline packages/auth/__tests__/hosted-ui.test.ts:161]: Is there a better way of doing this? I wanted to mock the `browserOrNode` within this module which is why I did `import *`

**Pull request merged**